usaspending_api/api_contracts/contracts/v2/search/spending_by_category.md

```diff
@@ -75,7 +75,7 @@ This endpoint returns a list of the top results of specific categories sorted by
 + `agencies` (optional, array[AgencyObject], fixed-type)
 + `recipient_search_text`: `Hampton` (optional, array[string])
 + `recipient_id` (optional, string)
```

Comment:
Should this also be nullable?

Reply:
Yes.  Good catch.  I'll check that in if/when I figure out how to fix this unit test.

Reply:
Oh, I'm sorry.  That's part of the request, right?  I don't think the request `recipient_id` is nullable, just the response - so really the mistake is really up above in the response.